README.md

```diff
@@ -118,6 +118,35 @@ in the middle of a session will lead to erroneous behaviors by LineaPy.
 - This loads the extension to the current session only, i.e., it does not carry over
 to different sessions; you will need to repeat it for each new session.
 
+#### Hosted Jupyter Environment
+
+In hosted Jupyter notebook environments such as JupyterHub, Google Colab, Kaggle or other environments that you do not start your notebook from CLI(such as Jupyter extension within VS Code), you need to install `lineapy` directly within your notebook first via:
+
+```ipython
+!pip install lineapy
+```
+
+then you can manually load `lineapy` with :
+
+```python
+%load_ext lineapy
+```
+
+For environments with older versions `IPython` like Google Colab, we need to upgrade the `IPython` module before the above steps, we can upgrade `IPython` via:
+
+```ipython
+%%capture

```

Comment:
remove `%%capture`. it was added to make the output look less messy but not necessary for personal use.